frontend/src/app/core/services/notifications.service.ts

```diff
@@ -33,12 +37,6 @@ export class NotificationsService implements OnDestroy {
     this.configureSignalR();
```

Comment:
What is the purpose of notifications$ ? Why don't you load all notifications directly in the place where it's needed?